tests/lib/cli-engine/_utils.js

```diff
@@ -54,7 +54,6 @@
 "use strict";
 
 // To use TypeScript type annotations for VSCode intellisense.
-/* eslint-disable valid-jsdoc */
```

Comment:
I'm glad that we get the supports of TypeScript type notations 😄 .

Reply:
Thanks for catching that. Updated!